src/themes/schulportal/login/resources/css/styles.css

```diff
@@ -0,0 +1,365 @@
+}
+
+.light-header {
+  background: #e5eaef;
+  height: 40px;
+  position: fixed;
+  top: 60px;
+  width: 100%;
+  z-index: 1006;
+}
+
+.login-pf-header {
+  font-size: 22px;
+  font-weight: bold;
+  max-width: 500px;
+  min-width: 320px;
+}
+
+.login-pf-page .login-pf-header h1 {
+  text-align: left;
+}
+
+.title-logo {
+  margin: 6.25rem auto 24px auto;
+  max-width: 600px;
+  text-align: center;
+  width: 540px;
+}
+
+.card-pf {
+  border-radius: 4px;
+  box-shadow: 0px 3px 10px #E5EAEF;
+  margin-left: auto;
+  margin-right: auto;
+  max-width: 540px;
+  min-width: 320px;
+}
+
+h1#kc-page-title {
+  font-size: 18px;
+  font-weight: bold;
+  margin-top: 0;
+}
+
+.card-divider {
+  background: #1EAE9C;
+  border-radius: 4px;
+  height: 4px;
+  margin-left: -40px;
+  margin-right: -40px;
+}
+
+.login-pf-page .card-pf p {
+  color: #001E49;
+}
+
+.login-prompt {
+  font-size: 16px;
+  line-height: 18px;
+  margin-top: 28px;
+  margin-bottom: 16px;
+}
+
+.pf-c-form__label.pf-c-form__label-text {
+  font-size: 16px;
+}
+
+.pf-c-input-group {
+  border: 1px solid #001E49;
+  border-radius: 5px;
+}
+
+.pf-c-input-group .pf-c-form-control {
+  border: none;
+}
+
+.pf-c-input-group:has(> .pf-c-form-control[aria-invalid=true]) {
+  border-color: #D4004B;
+}
+
+.pf-c-form-control:not(textarea) {
+  border: 1px solid #001E49;
+  border-radius: 5px;
+  height: 50px;
+  padding: 16px;
+  --pf-c-form-control--Color: #001E49;
+  --pf-c-form-control--placeholder--child--Color: #001E49;
+}
+
+.pf-c-form-control:focus {
+  padding-bottom: 16px;
+  border-bottom-width: 1px;
+}
+
+.pf-c-form-control:focus-visible {
+  outline-color: #001E49;
+}
+
+.pf-c-form-control:not(textarea)[aria-invalid=true] {
+  border-bottom-width: 1px;
+  border-color: #D4004B;
+  padding-bottom: 16px;
+  --pf-c-form-control--invalid--BackgroundUrl: url("data:image/svg+xml;charset=utf8,%3Csvg xmlns='http://www.w3.org/2000/svg' viewBox='0 0 512 512'%3E%3Cpath fill='%23D4004B' d='M504 256c0 136.997-111.043 248-248 248S8 392.997 8 256C8 119.083 119.043 8 256 8s248 111.083 248 248zm-248 50c-25.405 0-46 20.595-46 46s20.595 46 46 46 46-20.595 46-46-20.595-46-46-46zm-43.673-165.346l7.418 136c.347 6.364 5.609 11.346 11.982 11.346h48.546c6.373 0 11.635-4.982 11.982-11.346l7.418-136c.375-6.874-5.098-12.654-11.982-12.654h-63.383c-6.884 0-12.356 5.78-11.981 12.654z'/%3E%3C/svg%3E")
+}
+
+label:has(+ .pf-c-form-control:not(textarea)[aria-invalid=true]),
+label:has(+ .pf-c-input-group > .pf-c-form-control[aria-invalid=true]) {
+  color: #D4004B;
+  font-weight: bold;
+}
+
+.pf-c-button.pf-m-control {
+  --pf-c-button--BorderRadius: 5px;
+  --pf-c-button--after--BorderWidth: 0px;
+  border: none;
+}
+
+.pf-c-form__helper-text.pf-m-error {
+  color: #D4004B;
+}
+
+.password-help-text {
+  font-size: 12px;
+  text-align: center;
+}
+
+.pf-c-button.pf-m-primary {
+  background: #001E49;
+  border: 2px solid #001E49;
+  border-radius: 4px;
+  font-size: 16px;
+  height: 50px;
```

Comment:
Kann man hier auch die beiden styles hinzufügen? Dann funktioniert der Hover für Mobile wie in der Designvorlage:
    transition: transform 0.3s ease-in-out;
    transform-origin: left center;

Reply:
Ich würde es für Desktop hinzufügen, mobil gibt es ja eh keinen Hover.

Reply:
Ja die beiden Styles für `.pf-c-button.pf-m-primary` hinzufügen und es soll sich erledigen.

Reply:
done